### ISSUE TITLE
Allow for multiple strums in one segment.

### ISSUE THREAD

**Issue by bbroeksema** [syntagmatic/parallel-coordinates#69]
Currently only one strum can be set in each segment. However, it would give a lot of flexibility when the user can set two or more strums in the same segment.

**Comment by julianheinrich** [comment]
Conceptually, this is the same as #25

**Comment by bbroeksema** [comment]
@julianheinrich Yes, but technically completely different. I actually have some code lying around for multiple strums already, though just no time to actually integrate it.

No such starting point for vertical brushes (as this currently uses that d3 axes/brushing mechanism).

**Comment by julianheinrich** [comment]
@bbroeksema Ok thanks for letting me know. I'm currently working on a solution for #25.
Do you have time to get the strums done soon or can you share that code of yours?
